tests/test_bluez5.py

```diff
@@ -128,6 +128,24 @@ def test_one_adapter(self):
         self.assertIn("Discoverable: no", out)
         self.assertIn("Pairable: yes", out)
         self.assertIn("Discovering: no", out)
+        self.assertIn("Roles: central", out)
+        self.assertIn("Roles: peripheral", out)
+
+        # Advertising Manager
+        self.assertIn("Advertising Features:", out)
+        self.assertIn("ActiveInstances: 0x00 (0)", out)
+        self.assertIn("SupportedInstances: 0x05 (5)", out)
+        self.assertIn("SupportedIncludes: tx-power", out)
+        self.assertIn("SupportedIncludes: appearance", out)
+        self.assertIn("SupportedIncludes: local-name", out)
+        self.assertIn("SupportedSecondaryChannels: 1M", out)
+        self.assertIn("SupportedSecondaryChannels: 2M", out)
+        self.assertIn("SupportedCapabilities.MinTxPower: 0xffffffde (-34)", out)
+        self.assertIn("SupportedCapabilities.MaxTxPower: 0x0007 (7)", out)
+        self.assertIn("SupportedCapabilities.MaxAdvLen: 0xfb (251)", out)
+        self.assertIn("SupportedCapabilities.MaxScnRspLen: 0xfb (251)", out)
+        self.assertIn("SupportedFeatures: CanSetTxPower", out)
+        self.assertIn("SupportedFeatures: HardwareOffload", out)
```

Comment:
This also isn't in the [centos 9 output](https://github.com/martinpitt/python-dbusmock/actions/runs/9743502345/job/26898417289?pr=208#step:3:862) -- hmm, these are exactly the two you advertise in the mock. It works everywhere else, so this must be related to an older client version. Maybe you have an idea why, but if not, let's just make these conditional?

Reply:
Looks like `SupportedFeatures` was only added to the API with BlueZ 5.57 (CentOS 9 is on 5.56). I'll make this test conditional on the detected BlueZ version as well.

Reply:
Hopefully resolved now! :)